src/builders.ts

```diff
@@ -14,7 +14,7 @@ import { DaysToDecay, FilePath, LintMessage, LintResult, TodoData } from './type
 export function buildTodoData(
   baseDir: string,
   lintResults: LintResult[],
-  daysToDecay?: DaysToDecay
+  daysToDecay?: TodoConfig
```

Comment:
Any objection to changing the variable name also?

Reply:
Oh no, this was just a miss on my part.

Reply:
Fixed.